.github/workflows/publish-specs.yml

```diff
@@ -64,6 +65,7 @@ jobs:
             group: gpuwg
             status: ${{ matrix.w3c_build_override_status }}
 
+  # TODO: Is this step checking anything that the other steps/workflows aren't checking?
```

Comment:
This step might be better to preserve (I think there was an idea to isolate what happens to that step, I will recheck while adjusting multi-file support for WGSL).

Reply:
I think we can keep it for now, but maybe leave the TODO to investigate later?

Reply:
@kainino0x SGTM, thanks!